src/util/hasValue.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * ensures that the value passed in is not null or undefined
```

Comment:
sorry to be pedantic but it also ensures that it is not an empty string

Reply:
done